README.md

```diff
@@ -1,24 +1,69 @@
-# README
+# Hello React Rails
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This project is an implementation of connect react-rails gem. Built with Ruby, Javascript and Ruby on Rails
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails
+- JavaScript
+- CSS
 
-* System dependencies
+## Getting Started
 
-* Configuration
+```
+git clone https://github.com/wuyepabdul/hello-rails-backend
 
-* Database creation
+cd hello-rails-backend
+rails db:create
+rails db:migrate
+rails db:seed
 
-* Database initialization
+rails server
 
-* How to run the test suite
+```
 
-* Services (job queues, cache servers, search engines, etc.)
+## Live Demo
+- [View Demo](https://duls-react-rails-frontend.herokuapp.com/)
 
-* Deployment instructions
+## Front-end git repo
+```
+git clone https://github.com/wuyepabdul/hello-rails-frontend
```

Comment:
- [ ] [Optional] I can see this is a link to the frontend app, but you are still required to write it and highlight it and it looks like a link in order to find it so easily